firebase_admin/_user_mgt.py

```diff
@@ -492,6 +599,58 @@ def get_user(self, **kwargs):
                 http_response=http_resp)
         return body['users'][0]
 
+    def get_users(self, identifiers):
+        """Looks up multiple users by their identifiers (uid, email, etc.)
+
+        Args:
+            identifiers: UserIdentifier[]: The identifiers indicating the user
+                to be looked up. Must have <= 100 entries.
+
+        Returns:
+            list[dict[string, string]]: List of dicts representing the JSON
+            `UserInfo` responses from the server.
+
+        Raises:
+            ValueError: If any of the identifiers are invalid or if more than
+                100 identifiers are specified.
+            UnexpectedResponseError: If the backend server responds with an
+                unexpected message.
+        """
+        if not identifiers:
+            return []
+        if len(identifiers) > 100:
+            raise ValueError('`identifiers` parameter must have <= 100 entries.')
+
+        payload = {}
+        for identifier in identifiers:
+            if isinstance(identifier, _user_identifier.UidIdentifier):
+                _auth_utils.validate_uid(identifier.uid, required=True)
+                payload['localId'] = payload.get('localId', []) + [identifier.uid]
```

Comment:
.append()

Reply:
append doesn't work (directly) here. The problem is that if localId doesn't exist in the dict, then it returns an empty list. This list would then be appended with the uid, but since list.append() doesn't return itself, we end up setting payload['localId'] to None.

OTOH, using a defaultdict would allow append to work nicely. Done.

Reply:
Thanks. I like where this ended up 👍